minigeth/core/types/transaction_signing.go

```diff
@@ -182,6 +182,7 @@ func NewLondonSigner(chainId *big.Int) Signer {
 }
 
 func (s londonSigner) Sender(tx *Transaction) (common.Address, error) {
+	// TODO this also has to include the shutter types
```

Comment:
either I don't understand the comment or this is already done. `eip2930Signer` works for shutter and batch txs, no?

Reply:
Yes I'm sorry, this is a leftover